build.gradle.kts

```diff
@@ -140,6 +146,24 @@ application {
     mainClass.set("org.example.csv2tex.ui.Csv2TexApplication")
 }
 
+// ######################################### JaCoCo test coverage #########################################
+
+jacoco {
+    toolVersion = "0.8.7"
+}
+
+tasks.withType(Test::class.java) {
+    extensions.configure(JacocoTaskExtension::class) {
+        setDestinationFile(layout.buildDirectory.file("jacoco/test.exec").get().asFile)
+    }
+    finalizedBy(tasks.jacocoTestReport)
+}
+tasks.jacocoTestReport {
+    executionData.from(layout.buildDirectory.file("jacoco/test.exec"))
+}
```

Comment:
Hard-coding this so that all single test jobs in CircleCI will have an HTML report as output. 

I could configure this generically like so to find all .exec files: ...
```
tasks.jacocoTestReport {
    val fileNameFilter: (File, String) -> Boolean = { _: File, name: String -> name.endsWith(".exec") }
    executionData.from(layout.buildDirectory.dir("jacoco").map { it.asFile.listFiles(fileNameFilter) }.orNull)
}
```
..., but the report task gets configured before the tests generate the exec files as output. 
I.e., it would take a dedicated call to the jacoco task after the test task ran, or the jacoco task will not find any exec files on the first run.

Reply:
:information_source: I found something in a footnote here, it's now working without hard-coded file names

https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:up_to_date_checks